src/mail/model/MailUtils.ts

```diff
@@ -141,8 +141,7 @@ export function isTutanotaTeamMail(mail: Mail): boolean {
 
 /** this is used for contact form messages to prevent them from being sent to recipients that are not the contact forms target mail group */
 export function areParticipantsRestricted(mail: Mail): boolean {
-	const { restrictions } = mail
-	return restrictions != null && restrictions.participantGroupInfos.length > 0
+	return false
```

Comment:
so, what's with old mail that has this restriction? will they be able to be moved now?

also I feel like we can delete this function now that it's not doing anything

Reply:
Might've left that in by mistake since I was going to then remove all code paths that depend on the return value being true.

I'll clean it up in a bit.

Reply:
Fixed! This function was no longer being called anyway.